core/src/main/java/org/jobrunr/scheduling/RecurringJobBuilder.java

```diff
@@ -187,6 +188,32 @@ public RecurringJobBuilder withDuration(Duration duration) {
         return this;
     }
 
+    /**
+     * Allows to specify a {@link CronExpression} (preferably use {@link RecurringJobBuilder#withCron(String)}),
+     * an ISO-8601 duration (preferably use {@link RecurringJobBuilder#withDuration(Duration)})
+     * or a carbon aware schedule expression (see {@link CarbonAware}) that will be used to create the recurringJobs.
+     *
+     * <h5>Examples:</h5>
+     *
+     * <pre>{@code
+     *      aRecurringJob()
+     *          .withScheduleExpression(CarbonAware.dailyBefore(7))
+     *          .withDetails(() -> service.doWork());
+     * }</pre>
+     *
+     * <pre>{@code
+     *      aRecurringJob()
+     *          .withScheduleExpression("0 * * * * [PT0H/PT7H]")
```

Comment:
For me, this looks like an illegal CRON for the given Carbon Margin (a total duration of 7 hours) as the margin is larger than the CRON expression (every hour).

I think the Duration between cron expressions should always be smaller than the carbon aware margin.

Reply:
Good catch! Should be `0 0 * * *`, to match the previous example with `CarbonAware.dailyBefore(7)`